src/Components/Artwork/SaveButton/NewSaveButton.tsx

```diff
@@ -17,38 +17,20 @@ export interface SaveButtonProps {
   contextModule: AuthContextModule
 }
 
+interface NewSaveButtonBaseProps {
+  isSaved: boolean
+  onClick: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+}
+
 const BTN_HEIGHT = 20
 const BTN_WIDTH = 20
 
-export const NewSaveButton: React.FC<SaveButtonProps> = ({
-  artwork,
-  contextModule,
+export const NewSaveButtonBase: React.FC<NewSaveButtonBaseProps> = ({
```

Comment:
#non-blocking I know this code was already roughly here so no need to change in this PR, but is there any way we can avoid using "New" in the name, or rename old `SaveButton` to `DeprecatedSaveButton` and then this to `SaveButton`?

Reply:
Good suggestion. I'll rename it

Reply:
oh yeah, no prob at all! tagged it non-blocking but wanted to call it out 👍

Reply:
thanks!

Reply:
Addressed here: https://github.com/artsy/force/pull/11932